src/OrbitGl/CaptureWindow.cpp

```diff
@@ -72,6 +72,8 @@ class AccessibleCaptureWindow : public AccessibleWidgetBridge {
 using orbit_client_protos::TimerInfo;
 
 CaptureWindow::CaptureWindow(OrbitApp* app) : GlCanvas(), app_{app} {
+  capture_client_app_ = app;
```

Comment:
Nit: this could also be initialised in the initializer list.

Reply:
I would also prefer to have it in the member initializer list. BTW, is `app_` still needed?

Reply:
Yes, currently we need both - I wanted to emphasize a reasonable scope and responsibility for the interface instead of simply including everything needed in the capture window. There's still parts of `CaptureWindow` that will rely on `app_ != nullptr` and can't be called in unit tests for the time being, this needs to be improved in the future.